src/bls12_381/ec/g1.rs

```diff
@@ -193,6 +193,12 @@ impl fmt::Debug for G1Compressed {
     }
 }
 
+impl G1Compressed {
+    fn from_bytes_unchecked(original: [u8; 48]) -> Self {
```

Comment:
these need to be declared pub, otherwise nobody can use them. 
can you also add a doc string descring what this does please

Reply:
Done. Hope those docs are helpful. Let me know if something is missing or misleading.